lib/AdvancedPolls/Entity/Desc.php

```diff
@@ -57,9 +57,9 @@ class AdvancedPolls_Entity_Desc extends Zikula_EntityAccess
     /**
      * The following are annotations which define the description field.
      *
-     * @ORM\Column(type="text", nullable="true")
```

Comment:
Why did you remove it completely instead of removing the quotes only, like below?